packages/playground/website-deployment/mime-types.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+// Adapted from https://github.com/nginx/nginx/blob/d8a849ae3c99ee5ca82c9a06074761e937dac6d6/conf/mime.types
+$mime_types = array(
```

Comment:
Let's reuse this list in the PHPRequestHandler class

Reply:
@adamziel, sounds good.

My plan is to create a separate PR where we:
- add a mime-types JSON file
- import it in the TypeScript module for PHPRequestHandler
- use the JSON to generate mime-types.php during deployment